pyzip-demo-db-apb/roles/provision-pyzip-demo-db-apb/tasks/main.yml

```diff
@@ -114,9 +114,11 @@
     - zipcodes.sql
 
 - name: encode bind credentials
-  shell: 'echo "{\"POSTGRES_HOST\": \"{{ dbservice_name }}\", \"POSTGRES_PORT\": \"5432\", \"POSTGRES_USER\": \"{{ database_user }}\", \"POSTGRES_PASSWORD\": \"{{ database_password }}\", \"POSTGRES_DB\": \"{{ database_name }}\"}" | base64 -w 0'
-  register: encoded_bind_credentials
+  asb_encode_binding:
```

Comment:
Couldn't we also do this with an existing module? I found this one: http://docs.ansible.com/ansible/latest/blockinfile_module.html